File: src/main/java/br/com/jiratorio/repository/UserConfigRepository.java
package br.com.jiratorio.repository;

import br.com.jiratorio.domain.entity.UserConfig;
import java.util.Optional;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserConfigRepository extends CrudRepository<UserConfig, Long> {

    Optional<UserConfig> findByUsername(String username);

}

File: src/main/kotlin/br/com/jiratorio/repository/UserConfigRepository.kt
package br.com.jiratorio.repository

import br.com.jiratorio.domain.entity.UserConfig
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.Optional

@Repository
interface UserConfigRepository : CrudRepository<UserConfig, Long> {

    fun findByUsername(username: String): Optional<UserConfig>

}
